README.md

```
@@ -155,10 +155,23 @@ Auto-classifies a GitHub issue into a tier, then spawns an agent with a tier-app
 /solve 123 --small                    # force small tier
 /solve 123 --full                     # force medium/large
 /solve 123 --auto                     # enable --dangerously-skip-permissions (bypass all permission gates)
-claude agents                          # monitor active /solve and /turbo background sessions
+/workflows                             # monitor active /solve and /turbo solver fleets
 ```
 
-**What runs as the background session** — model and effort are pinned so behavior is reproducible across runs:
+**What actually runs** (RFC 0015) — the launcher validates, triages and claims every issue, then hands the batch to `skills/solve-fleet/workflow.js`. That script runs **one solver agent per issue in its own git worktree**, in `parallel()` waves of `fanout_concurrency.solve_bg` (default 6):
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)